lredis/commands.lua

```diff
@@ -40,6 +41,48 @@ function methods:client_pause(delay)
 	return handle_ok_or_err(resp)
 end
 
+function methods:hmget(key, ...)
+	local resp = self:call("HMGET", key, ...)
+	if type(resp) == "table" then
+		local ret = {}
+		for i, v in pairs({...}) do
```

Comment:
Don't use `pairs` to iterate an array: at least use `ipairs`. But better: use `table.pack` and the `.n` field.